src/current/v23.1/role-based-audit-logging.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Role-based SQL Audit Logging
+summary: Use the sql.log.user_audit cluster setting to turn SQL audit logging on or off for a user or role.
+toc: true
+docs_area: manage
+---
+
+Role-based SQL audit logging gives you detailed information about queries being executed against your system by specific users or roles. An event of type [`role_based_audit_event`]({% link {{ page.version.version }}/eventlog.md %}#role_based_audit_event) is recorded when an executed query belongs to a user whose role membership corresponds to a role that is enabled to emit an audit log via the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit). The event is logged in the [`SENSITIVE_ACCESS`]({% link {{ page.version.version }}/logging.md %}#sensitive_access) logging channel.
+
+{% include enterprise-feature.md %}
+
+This page shows how to enable role-based SQL audit logging in CockroachDB, including:
+
+- How to configure audit logging.
+- Where the audit log files live.
+- What the audit log files look like.
+
+{{site.data.alerts.callout_info}}
+Logging has performance implications. This feature enables you to log only what you require allowing you to limit logging and reduce impact to your workload.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For the best visibility into security-related events on your cluster, we recommend configuring `SENSITIVE_ACCESS` together with the `USER_ADMIN`, `PRIVILEGES`, and `SESSIONS` logging channels. To learn more, see [Logging Use Cases]({% link {{ page.version.version }}/logging-use-cases.md %}#security-and-audit-monitoring).
+{{site.data.alerts.end}}
+
+## Configure Logging
+
+### Prerequisites
+
+You must have:
+
+- [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or `MODIFYCLUSTERSETTING` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) to configure the cluster setting,
+- an [Enterprise license configured]({% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license), and
+- your [logging infrastructure configured]({% link {{ page.version.version }}/configure-logs.md %}), particularly the [location of audit logs](#file-storage-location).
+
+### Syntax of audit settings
+
+With the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit), you can set an audit logging configuration using a table-like syntax, as seen in the [Examples](#examples). Each row in the configuration represents an audit setting and must be separated by a new line. An audit setting is comprised of two columns separated by a space:
```

Comment:
```suggestion
With the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit), you can set an audit logging configuration using a table-like syntax, as shown in the [Examples](#examples). Each row in the configuration represents an audit setting and must be separated by a new line. An audit setting is comprised of two columns separated by a space:
```
style guide nit, try to avoid "see/seen"

Reply:
updated